### ISSUE TITLE
What is the meaning of this parameter n_critic？

### ISSUE THREAD

**Issue by optimalSolution-Q** [daa233/generative-inpainting-pytorch#32]
Why is it necessary to calculate the loss of the generator every five times instead of calculating it every time
![image](https://user-images.githubusercontent.com/41374112/79586530-d2382000-8103-11ea-8124-3572c476e8a6.png)

**Comment by daa233** [comment]
Q: What is the meaning of this parameter n_critic?
A: It means 'the number of iterations of the critic (discriminator) per generator iteration'. It was originally proposed by the WGAN paper, and the WGAN-GP paper (which is used in this repo) follows this practice.

Q: Why is it necessary to calculate the loss of the generator every five times instead of calculating it every time?
A: Because with WGAN-series loss, we can (and should) train the critic till optimality. It means training the critic more will get better results. However, training the critic more will slow down the convergence. That's why the hyperparameter n_critic is 5 here.

**Comment by optimalSolution-Q** [comment]
Can you add me on WeChat? I still have some questions about the code, the QR code has been sent to your Gmail email(from you github homepage)

**Comment by daa233** [comment]
Sorry, I don't like answer questions about the code on WeChat. You can ask here or by email.
